packages/eas-cli/src/build/build.ts

```diff
@@ -486,8 +486,8 @@ async function handleSingleBuildProgressAsync(
       spinner.succeed('Build finished');
       return { refetch: false };
     case BuildStatus.New:
-      spinner.text = `Build is waiting to enter the queue. Check your concurrency limit at ${link(
-        formatAccountSubscriptionsUrl(accountName)
+      spinner.text = `Build concurrency limit reached for your account, waiting for a concurrency to become available. Add additional concurrencies at ${link(
```

Comment:
```suggestion
      spinner.text = `Build concurrency limit reached for your account. Build will enter queue once a concurrency becomes available. Add additional concurrencies at ${link(
```
Just an idea... I think the original message communicates slightly better what's happening to the _build_.